### PR TITLE
Adding raster tutorial contents

### PR DESCRIPTION
**Pull request by phargogh**
Includes:

* Dockerfile
* `environment.yml`
* data
* notebooks

### CODE REVIEW

**Comment by lsetiawan** [comment]
@phargogh Is this ready for merging?

File: raster/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM continuumio/anaconda
```
Comment (lsetiawan) [inline raster/Dockerfile]: We don't need dockerfile this year. Please remove to keep repository clean. Thanks! :)

**Comment by lsetiawan** [comment]
Thanks @phargogh. Okay for merging?

**Comment by phargogh** [comment]
Yep, should be good to go.  Thanks!

**Pull request merged**